src/gpu/intel/ocl/micro_sdpa.cpp

```diff
@@ -413,7 +413,11 @@ status_t micro_sdpa_t::init(impl::engine_t *engine) {
     auto ldk = gemm_desc_t::get_ld(*pd()->key_md()) * key_mdw.data_type_size();
     auto ldv = gemm_desc_t::get_ld(*pd()->val_md()) * val_mdw.data_type_size();
     auto lda = gemm_desc_t::get_ld(*pd()->dst_md()) * dst_mdw.data_type_size();
-    auto ldmsk = pd()->attn_mask_md()->dims[3] * msk_mdw.data_type_size();
+    // If without attn mask, msk_mdw.data_type_size() will hit assertion in
```

Comment:
I suggest you remove the comment here. It doesn't add much to the context.

Reply:
Thank you for the quick review, Umar. It's removed now.